source/includes/connection-snippets/x509-snippets/php-connection.php

```diff
@@ -0,0 +1,13 @@
+<!-- Begin x509 connection -->
+<?php
+$subject = "mongodb+srv://<your_subject>"; 
+$manager = new MongoDB\Driver\Manager(
+    $subject + "@<cluster-url>/?ssl=true&authMechanism=MONGODB-X509",
```

Comment:
`+` is not a string concat operator in PHP. This should be `.` if so; however, I don't think you actually need to store the first part of the URI in a `$subject` variable.